features/step_definitions/basic_step.rb

```diff
@@ -1,16 +1,45 @@
-Then(/^show me the page$/) do
-  save_and_open_page
-end
-
+#Action steps
 When(/^I visit "([^"]*)" page$/) do |page_name|
   visit get_path page_name
 end
 
+#Form action steps
+When(/^I click "([^"]*)" (?:link|button)$/) do |link_or_button_name|
+  click_link_or_button link_or_button_name
+end
+
+When(/^I fill in "([^"]*)" with "([^"]*)"$/) do |field, value|
+  fill_in field, with: value
+end
+
+Given(/^I select "([^"]*)" from the "([^"]*)"$/) do |option, select_field|
+  select option, from: select_field
+end
+
+#Assertions steps
+Then(/^I should see "([^"]*)"$/) do |content|
+  expect(page).to have_content content
+end
+
 Then(/^I should be on "([^"]*)" page$/) do |page_name|
   expect(page).to have_current_path get_path page_name
```

Comment:
A bit out of scope for this PR but I want to comment on this. This is not readable. I would actually split this into two lines. 
```ruby 
path = get_path(page_name)
expect(page).to have_current_path path
```
Since you are calling an method to get the path I would argue that this is more readable.